geomesa-tools/examples/ingest/example1.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+geomesa ingest -u user -p password -i inst -z zoo -c catalog -conf example1.conf example1.csv
```

Comment:
Eh, looking at the other commands we're not very consistent anywhere - just go with whatever.